app/Tquery/Config/TqConfig.php

```diff
@@ -61,6 +61,17 @@ public function addQuery(
         );
     }
 
+    /** Added in TqRequest for "distinct" queries */
+    public function addCount(): void
+    {
+        $this->addColumn(
+            type: TqDataTypeEnum::int,
+            columnOrQuery: fn(string $tableName) => 'count(1)',
```

Comment:
To jest coś inne niż `count(*)`?

Reply:
takie samo jest, ale mi się bardziej podoba